contracts/hydro/src/contract.rs

```diff
@@ -90,6 +91,10 @@ pub const MIN_DEPLOYMENT_DURATION: u64 = 1;
 
 pub const MIN_SPLIT_LOCK_SIZE: Uint128 = Uint128::new(10_000);
 
+pub const LOCK_EXPIRY_DURATION_SECONDS: u64 = 60 * 60 * 24 * 30; // 30 days
```

Comment:
I think both of these constants should be moved into `Constants` so they can be updated without need to store the new codes and perform migration.

Also the `update_config()` should be modified to support update of those values.

Keep in mind that this change also requires migration of Constants in our deployed contracts.